### DISCUSSION TITLE
Adding Glass Break Sensors without Ring Alarm Base Station

### DISCUSSION THREAD

**Discussion by allexvalentin** [tsightler/ring-mqtt#794] [Support Q&A] [answered]
Hello.
I am new to HA and was looking to add Ring's Glass Break sensors only to my existing setup. Is there a way to pair a brand new glass break sensor and get it to show in HA without having the base station?
I really appreciate your help with this.
Thanks in advance,
Allex

**Comment by tsightler** [comment, marked as answer]
Ring sensors are just Z-wave devices so you can manually pair them with pretty much any Z-wave hub or USB stick, however, that would have nothing to do with this project.

**Reply by allexvalentin** [reply]
Thank you so much for the guidance. I apologize for including it here. I did not know it was not related to this project. I really appreciate your help though. Now I know I have to research where to get a Z-wave hub. 

Thanks again!
